apps/docs/content/guides/functions/ephemeral-storage.mdx

```diff
@@ -0,0 +1,48 @@
+---
+id: 'function-ephemeral-storage'
+title: 'Ephemeral Storage'
+description: 'Read and write from temporary directory'
+subtitle: 'Read and write from temporary directory'
+---
+
+Edge Functions provides ephemeral file storage. You can read and write files to `/tmp` directory.
+
+Ephemeral storage will reset on each function invocation. This means the files you write during an invocation can only be read only within the same invocation.
```

Comment:
if you have time, it could be worthwhile to point out the use-cases where this is useful and not useful

Reply:
good call! added https://github.com/supabase/supabase/pull/30791/commits/7d046aa9273e0341f940fe6f23b80997edbb1446